internal/utils/aggregator.go

```diff
@@ -104,27 +100,48 @@ func filter_valid_csv_records(main_df dataframe.DataFrame) (dataframe.DataFrame,
 		dataframe.F{Colname: "owner_name", Comparator: series.Neq, Comparando: ""},
 		dataframe.F{Colname: "owner_kind", Comparator: series.Neq, Comparando: "<none>"},
 		dataframe.F{Colname: "owner_name", Comparator: series.Neq, Comparando: "<none>"},
+		dataframe.F{Colname: "workload_type", Comparator: series.Neq, Comparando: "<none>"},
+		dataframe.F{Colname: "workload_type", Comparator: series.Neq, Comparando: ""},
 	)
 
-	no_of_dropped_records := main_df.Nrow() - df.Nrow()
+	// The above filters can delete all the rows
+	if df.Nrow() == 0 {
```

Comment:
@saltgen writing more tests unearth this situation, added a code for same!

Reply:
Good Stuff! 🌟